portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+    <!-- Icons from W3-->
+	<link rel="stylesheet" href="style.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<nav id="nav-bar">
+		<div class="drop-down">
+			<button onclick="photoDropdown()" id="media-dropdown"><img src="/images/me.jpg" id="menu-img" style="width:70px;height:70px">
+                <i class="fa fa-caret-down" style="color:#f0eded"></i>
+        </div>
+        <div id="drop-down-container">
+            <a href="https://www.linkedin.com/in/efrain-retana?trk=people-guest_people_search-card" class='dropdown-a'><i class="fa fa-linkedin"></i></a>
+            <a href="https://github.com/ERetana2" class='dropdown-a'><i class="fa fa-github"></i> </a>
+        </div>
+            <a href="index.html"><i class="fa fa-fw fa-home"></i> Home</a>
+            <a href="#"><i class="fa fa-fw fa-envelope"></i> Contact</a>
+            <a href="resume.html"><i class="material-icons">computer</i> Resume</a>
+    </nav>
+    <div id="content">
+		<h1 class="main-header" >Contact Info <p id="welcome-msg">Feel free to Reach out.</p> </h1>
+        <form class="contact-container" action ="/data" method="POST">
+
+            <label for="fname">First Name:</label>
+            <input type="text" id="fname" name="firstname" placeholder="First name..">
+
+            <label for="lname">Last Name:</label>
+            <input type="text" id="lname" name="lastname" placeholder="Last name..">
+
+            <label for="email">E-mail:</label>
+            <input type="text" id="email" name="email" placeholder="Enter e-mail..">
+
+            <label for="subject">Subject</label>
+            <textarea id="subject" name="subject" placeholder="Write something.." style="height:200px"></textarea>
+
+            <input type="submit" value="Submit">
+
+        </form>
+    </div>
+
+</body>
+<footer>
+    <p>Copyright 2020 &copy; by Efrain Retana </p>
+
+</footer>
```

Comment:
add new line at end of file to fix this warning. also add new line to other html files with this warning